### PR TITLE
Add EntityEatEvent

### PR DESCRIPTION
**Pull request by tips48**
Currently called when a sheep eats a grass block. Tested to make sure it works, contact me if you need the test plugin source

CraftBukkit pull: https://github.com/Bukkit/CraftBukkit/pull/633

Relates to BUKKIT-512

### CODE REVIEW

File: src/main/java/org/bukkit/event/entity/EntityEatEvent.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import org.bukkit.event.Cancellable;
+
+@SuppressWarnings("serial")
+public class EntityEatEvent extends EntityEvent implements Cancellable {
+    
+    private boolean cancel;
+    private Block block;
+    
+    public EntityEatEvent(Entity what, Block block) {
+	super(Type.ENTITY_EAT, what);
```
Comment (Dinnerbone) [inline src/main/java/org/bukkit/event/entity/EntityEatEvent.java]: Tabs!

Comment (tips48) [inline src/main/java/org/bukkit/event/entity/EntityEatEvent.java]: D: Eclipse failed me! One sec :/

**Comment by Dinnerbone** [comment]
And now it's not indented fully. :D

**Comment by iffa** [comment]
heh

**Comment by tips48** [comment]
Arghh! I'll get it done..:/

**Comment by tips48** [comment]
I'm going to rewrite because rebasing didn't work -.-

**Pull request closed without merging**